### DISCUSSION TITLE
Why does useToken not return Token Name?

### DISCUSSION THREAD

**Discussion by layinka** [wevm/wagmi#187] [Q&A]
Hi,

Why does useToken hook not return Token name,it only returns token symbol.

It should return name too

**Comment by tmm** [comment]
Didn't seem as widely used as symbol in interfaces. Can consider adding this in a future version.

**Comment by layinka** [comment]
Really? It should still be there, regardless-------- Original message --------From: awkweb ***@***.***>Date: Sat, 19 Feb 2022, 23:29To: tmm/wagmi ***@***.***>Cc: layinka ***@***.***>, Author ***@***.***>Subject: Re: [tmm/wagmi] Why does useToken not return Token Name? (Discussion #187)
Didn't seem as widely used as symbol in interfaces.
